tests/ts_simple_select.erl

```diff
@@ -27,14 +27,18 @@
 -export([confirm/0]).
 
 confirm() ->
-    TestType = normal,
-    DDL = ts_util:get_ddl(),
-    Data = ts_util:get_valid_select_data(),
-    Qry = ts_util:get_valid_qry(),
+    Table = ts_data:get_default_bucket(),
+    DDL = ts_data:get_ddl(),
+    Data = ts_data:get_valid_select_data(),
+    Qry = ts_data:get_valid_qry(),
     Expected =
-        {ok, {ts_util:get_cols(small),
-         ts_util:exclusive_result_from_data(Data, 2, 9)}},
-    Got = ts_util:ts_query(
-            ts_util:cluster_and_connect(single), TestType, DDL, Data, Qry),
+        {ok, {ts_data:get_cols(small),
+         ts_data:exclusive_result_from_data(Data, 2, 9)}},
+
+    Cluster = ts_setup:start_cluster(1),
+    ts_setup:create_bucket_type(Cluster, DDL, Table),
+    ts_setup:activate_bucket_type(Cluster, Table),
+    ts_ops:put(Cluster, Table, Data),
+    Got = ts_ops:query(Cluster, Qry),
     ?assertEqual(Expected, Got),
```

Comment:
If you add up to the time we spent converting tests to use row containers from lists to tuples, I doubt it was a lot, and now it is over. If I added up the amount of time I spend trying to work out what some of our tests are doing it would be much longer! And I pay it every time I need to read the tests.  Tests are read heavy code, we should optimise them to be read in isolation as opposed to a part of the bigger framework.  This is the opposite of how I would write most production code of course.

This is one of those sweeping changes, but it doesn't affect those tests that are isolated. That don't use the utility functions and have shared data.  Re-usability also has costs.